manifests/08-kube-system-rbac.yaml

```diff
@@ -1,13 +1,30 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: coreos-pull-secret-reader
+  namespace: kube-system
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
+
+---
+
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: cluster-samples-operator-kube-system-edit
+  name: cluster-samples-operator-kube-system-view
```

Comment:
cluster-samples-operator-kube-system-secret-reader ?

Reply:
sure ... consistency

part of next push